File: 015/Problem15.mjs
import { performance } from "perf_hooks";

function problem() {
  var gridSize = 20;
  var paths = 1;
  for (var i = 0; i < gridSize; i++) {
    paths *= 2 * gridSize - i;
    paths /= i + 1;
  }
  console.log(paths);
}

var start = performance.now();
problem();
var end = performance.now();
console.log("Function took " + (end - start) / 1000);

File: 015/Problem15.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace EulerProblems
{
    class Program
    {
        static void Main(string[] args)
        {
            var watch = new System.Diagnostics.Stopwatch();

            watch.Start();
            Problem();
            watch.Stop();

            Console.WriteLine($"Function Took: {watch.Elapsed}");
        }

        private static void Problem()
        {
            var gridSize = 20;
            long paths = 1;
            for (var i = 0; i < gridSize; i++)
            {
                paths *= (2 * gridSize) - i;
                paths /= i + 1;
            }
            Console.WriteLine(paths);
        }
    }
}